BeeSwift/SceneDelegate.swift

```diff
@@ -1,87 +1,92 @@
 // Part of BeeSwift. Copyright Beeminder
 
+import BeeKit
 import CoreSpotlight
 import Foundation
 import OSLog
 import UIKit
 
-import BeeKit
-
 class SceneDelegate: UIResponder, UIWindowSceneDelegate {
-    let logger = Logger(subsystem: "com.beeminder.beeminder", category: "SceneDelegate")
-    
-    var window: UIWindow?
-    
-    private var coordinator: MainCoordinator?
-
-    func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        guard let windowScene = (scene as? UIWindowScene) else { return }
-        windowScene.requestGeometryUpdate(.iOS(interfaceOrientations: .all))
-
-        let navigationController = UINavigationController()
-        self.coordinator = MainCoordinator(
-            navigationController: navigationController,
-            currentUserManager: ServiceLocator.currentUserManager,
-            viewContext: ServiceLocator.persistentContainer.viewContext,
-            versionManager: ServiceLocator.versionManager,
-            goalManager: ServiceLocator.goalManager,
-            healthStoreManager: ServiceLocator.healthStoreManager,
-            requestManager: ServiceLocator.requestManager
-        )
-        
-        window = UIWindow(windowScene: windowScene)
-        window?.rootViewController = navigationController
-        window?.makeKeyAndVisible()
-        
-        coordinator?.start()
+  let logger = Logger(subsystem: "com.beeminder.beeminder", category: "SceneDelegate")
+
+  var window: UIWindow?
+
+  private var coordinator: MainCoordinator?
+
+  func scene(
+    _ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions
+  ) {
+    guard let windowScene = (scene as? UIWindowScene) else { return }
+    windowScene.requestGeometryUpdate(.iOS(interfaceOrientations: .all))
+
+    let navigationController = UINavigationController()
+    self.coordinator = MainCoordinator(
+      navigationController: navigationController,
+      currentUserManager: ServiceLocator.currentUserManager,
+      viewContext: ServiceLocator.persistentContainer.viewContext,
+      versionManager: ServiceLocator.versionManager,
+      goalManager: ServiceLocator.goalManager,
+      healthStoreManager: ServiceLocator.healthStoreManager,
+      requestManager: ServiceLocator.requestManager
+    )
+
+    window = UIWindow(windowScene: windowScene)
+    window?.rootViewController = navigationController
+    window?.makeKeyAndVisible()
+
+    coordinator?.start()
+  }
+
+  func scene(_ scene: UIScene, continue userActivity: NSUserActivity) {
+    logger.info("\(#function)")
+
+    var userInfoOfGoalFromSpotlight: [AnyHashable: Any]? {
+      guard userActivity.activityType == CSSearchableItemActionType else { return nil }
+      guard let goalIdentifier = userActivity.userInfo?[CSSearchableItemActivityIdentifier] as? String else {
```

Comment:
And here we _dont_ wrap the else, which is annoyingly inconsistent with the above case where we did.

Reply:
It appears it can be forced by setting [linebreakbeforecontrolflowkeywords](https://github.com/swiftlang/swift-format/blob/main/Documentation/Configuration.md#linebreakbeforecontrolflowkeywords) to `true`.